sonar-java-plugin/src/main/java/org/sonar/plugins/java/XmlFileSensor.java

```diff
@@ -43,22 +42,28 @@ public XmlFileSensor(SonarComponents sonarComponents, FileSystem fs) {
   }
 
   @Override
-  public boolean shouldExecuteOnProject(Project project) {
-    return fs.hasFiles(xmlFilePredicate);
+  public String toString() {
```

Comment:
The `toString()` method was already there (see line 61). I just moved it. As in `JavaSquidSensor` we kept it, I did the same here.